### PR TITLE
Added metaTitle generator to jsdoc-convert tool.

### PR DESCRIPTION
**Pull request by Siemienik**
### Context
I added a mechanism to generate `metaTitle`, which has been added manually.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. resolve https://github.com/handsontable/handsontable/issues/8188
2.
3.

### Affected project(s):
- [ ] `handsontable`
- [ ] `@handsontable/angular`
- [ ] `@handsontable/react`
- [ ] `@handsontable/vue`

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [x] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by Siemienik** [comment]
Blocked by: https://github.com/handsontable/handsontable/pull/8199
After that PR will be merge, I have to regenerate these docs again.

**Review by wszymanski — CHANGES_REQUESTED** [review]
Beside of mentioned above blocker, please fix introduced problems found by linter.

**Comment by Siemienik** [comment]
Closed as merged with #8199

**Pull request closed without merging**
